tests/voq/test_fabric_reach.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import yaml
+
+logger = logging.getLogger(__name__)
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+supReferenceData = {}
+localModule = 0
+supervisorAsicBase = 300
+
+# Added a function to setup the reference data for sup.
+
+
+def test_setup_reference_data(duthosts):
+    # supReferenceData has the expected data for sup
+    global supReferenceData
+    keys = []
+    if len(duthosts.supervisor_nodes) == 0:
+        logger.info("Please run the test on modular systems")
+        return
+    duthost = duthosts.supervisor_nodes[0]
+    logger.info("duthost: {}".format(duthost.hostname))
+    num_asics = duthost.num_asics()
+    logger.info("num_asics: {}".format(num_asics))
+    for asic in range(num_asics):
+        keys.append('asic' + str(asic))
+    supReferenceData = {key: {} for key in keys}
+
+# This test checks the output of the "show fabric reachability" command
+# on one linecard. It is called once for each linecard in the chassis.
+# It loads the reference data for the linecard, runs the CLI command,
+# and compares the output.
+
+
+def test_fabric_reach_linecards(duthosts, enum_frontend_dut_hostname):
+    """compare the CLI output with the reference data"""
+    global supReferenceData
+    global localModule
+    global supervisorAsicBase
+
+    # supReferenceData has the expected data
+    if len(duthosts.supervisor_nodes) == 0:
+        logger.info("Please run the test on modular systems")
+        return
+    duthost = duthosts.supervisor_nodes[0]
+    logger.info("duthost: {}".format(duthost.hostname))
+
+    # Get hwSku for Fabriccards from the supervisor.
+    fabric_sku = None
+    fabric_sku = duthost.facts['hwsku']
+    pytest_assert(fabric_sku, "Need to add hwSku information for sup")
+
+    # Load the reference data file.
+    duthost = duthosts[enum_frontend_dut_hostname]
+    logger.info("duthost: {}".format(duthost.hostname))
+    lc_sku = duthost.facts['hwsku']
+    slot = duthost.facts['slot_num']
+    fileName = lc_sku + "_" + fabric_sku + "_" + "LC" + str(slot) + ".yaml"
```

Comment:
Trying to run this against Nokia chassis. lc_sku and fabric_sku are the same value (duthosts.facts['hwsku']). Is that expected?